addons/web_editor/models/ir_ui_view.py

```diff
@@ -49,10 +49,16 @@ def save_embedded_field(self, el):
 
         if value is not None:
             # TODO: batch writes?
+            record = Model.browse(int(el.get('data-oe-id')))
             if not self.env.context.get('lang') and self.get_default_lang_code():
-                Model.browse(int(el.get('data-oe-id'))).with_context(lang=self.get_default_lang_code()).write({field: value})
+                record.with_context(lang=self.get_default_lang_code()).write({field: value})
             else:
-                Model.browse(int(el.get('data-oe-id'))).write({field: value})
+                record.write({field: value})
+
+            # copy_translations is not able to copy translations from one model
+            # field to another model field (yet?)
+            # if callable(Model._fields[field].translate):
+                # self._copy_custom_snippet_translations(record, field)
```

Comment:
@HydrionBurst As discussed, here I ideally should be able to tell to the `copy_translations()` (or any new method) which field's translations from model A I want to copy into another model B's field translations.

Like being able to copy the translations of an `ir.ui.view.arch_db` into a `product.template.product_description`.